src/file_logger/file_logger.go

```diff
@@ -0,0 +1,91 @@
+package file_logger
+
+import (
+	"time"
+
+	"github.com/HyperloopUPV-H8/Backend-H8/common"
+	"github.com/HyperloopUPV-H8/Backend-H8/logger_handler"
+	"github.com/rs/zerolog"
+	trace "github.com/rs/zerolog/log"
+)
+
+type FileLogger struct {
+	ids           common.Set[string]
+	fileName      string
```

Comment:
Thats why I stored them as properties instead of as part of the config, because flushInterval has no use

Reply:
I would store it as well because that way it is easier to expand the configuration, but we can also change it once we need it. If you feel like it is not necessary leave it like it is